runtimes/deno-runtime/README.md

```diff
@@ -29,7 +29,7 @@ console.log(secret);
 Replace `${VERSION}` with the KLT version of your choice.
 
 ```sh
-docker run -v $(pwd)/test.ts:/test.ts -e SCRIPT=/test.ts -e DATA='{ "url":"http://localhost:9090" }' -e SECURE_DATA='{ "token": "myToken"}' -it ghcr.io/keptn/functions-runtime:${VERSION}
+docker run -v $(pwd)/test.ts:/test.ts -e SCRIPT=/test.ts -e DATA='{ "url":"http://localhost:9090" }' -e SECURE_DATA='{ "token": "myToken"}' -it ghcr.io/keptn/deno-runtime:${VERSION}
 ```
 
```

Comment:
line 39 48 58 68path to the sample changed

Reply:
✔️